sanic_validation/decorators.py

```diff
@@ -18,6 +18,23 @@ def v(request, *args, **kwargs):
     return vd
 
 
+def validate_args(schema):
+    validator = Validator(schema)
+
+    def vd(f):
+        def v(request, *args, **kwargs):
+            normalized_args = validator.normalized(request.raw_args, schema)
```

Comment:
Is explicit normalization really necessary? From cerberus' docs:
```
validate(document, schema=None, update=False, normalize=True)
Normalizes and validates a mapping against a validation-schema of defined rules.
```

Reply:
Apparently it isn't. I haven't used Cerberus before.
TBCFH I have no real idea what normalize actually does.

But I'll remove it since it is apparently done by `validate`.